.NET/Microsoft.Recognizers.Text.DateTime.Tests/English/TestDatePeriodExtractor.cs

```diff
@@ -181,6 +181,7 @@ public void TestDatePeriodExtractSeason()
         public void TestDatePeriodExtractWeekOf()
         {
             //test week of and month of
+            BasicTest("the week of september.15th", 0, 26);

```

Comment:
In new unit tests we should add longer more complete sentences. That will help make sure corner cases don't happen.